HOC/WrapperProjectDetails.tsx

```diff
@@ -6,9 +6,9 @@ const WrapperProjectDetails = ({
 }: WrapperInterface) => {
   return (
     <div
-      className={`flex w-full flex-col items-center justify-center px-[1.5rem] py-[2.625rem] md:px-[5rem] ${additionalStyles}`}
+      className={`flex w-full flex-col items-center justify-center overflow-hidden px-[1.5rem] py-[2.625rem] md:px-[5rem]  ${additionalStyles}`}
```

Comment:
I would again recommend you rename additional styles to "className" and simply pass it on. It would make the usage more clear in code- as what you're doing is applying classNames.